src/DynamicRupture/FrictionLaws/LinearSlipWeakening.cpp

```diff
@@ -10,27 +17,19 @@ void BiMaterialFault::copyLtsTreeToLocal(seissol::initializers::Layer& layerData
   regularisedStrength = layerData.var(concreteLts->regularisedStrength);
 }
 
-void BiMaterialFault::strengthHook(real& strength,
+real BiMaterialFault::strengthHook(real& faultStrength,
                                    real& localSlipRate,
-                                   real& sigma,
-                                   real& mu,
                                    real& deltaT,
```

Comment:
Why do we need to pass it by-reference? It is going to be the same if we do it by-value